core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -124,61 +124,82 @@ export class ContributorAdminStatsTable implements OnInit {
     return num === 1 ? word : word + 's';
   }
 
-  getCountDisplay(
+  getEditTextForAttribute(
+      editCount: number,
+      contributionSubType: string): string {
+    return ' (' + editCount + ' ' +
+           (
+           contributionSubType ===
+           AppConstants.CONTRIBUTION_STATS_SUBTYPE_REVIEW ?
+           'edited' : 'without edits'
+           ) +
+           ')';
+  }
+
+  getTextForAttribute(
       count: number,
       wordCount: number = 0,
-      editDisplay: string = ''): string {
+      showEditText: boolean = false,
+      editCount: number = 0,
+      contributionSubType: string = ''): string {
     return count + this.pluralise(count, ' card') +
-           editDisplay +
          (
```

Comment:
You can move this to the end of the previous line.

Reply:
I've changed this a little, so there is a bracket here now, so I left it,